packages/core/admin/admin/src/components/MainNav/NavBrand.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, VisuallyHidden } from '@strapi/design-system';
+import { useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import { useConfiguration } from '../../features/Configuration';
+
+const BrandIconWrapper = styled(Box)`
+  svg,
+  img {
+    border-radius: ${({ theme }) => theme.borderRadius};
+    object-fit: contain;
+    height: ${24 / 16}rem;
+    width: ${24 / 16}rem;
+    margin: ${3 / 16}rem;
+  }
+`;
+
+export const NavBrand = () => {
+  const { formatMessage } = useIntl();
+  const {
+    logos: { menu },
+  } = useConfiguration('LeftMenu');
+  return (
+    <Box paddingLeft={3} paddingRight={3} paddingTop={3} paddingBottom={3}>
```

Comment:
I will just ask probably dumb questions, so feel free to correct me as much as you want

Does the box compo has a "padding" attribute to do something like <Box padding={3}> ?